diff --git a/Eigen/src/misc/lapacke.h b/Eigen/src/misc/lapacke.h
index 3d8e24f5a..f35ff8b24 100755
--- a/Eigen/src/misc/lapacke.h
+++ b/Eigen/src/misc/lapacke.h
@@ -73,7 +73,7 @@
 /* Complex type (single precision) */
 #ifndef lapack_complex_float
 #include <complex.h>
-#define lapack_complex_float    float _Complex
+#define lapack_complex_float    std::complex<float>
 #endif
 
 #ifndef lapack_complex_float_real
@@ -89,7 +89,7 @@ lapack_complex_float lapack_make_complex_float( float re, float im );
 /* Complex type (double precision) */
 #ifndef lapack_complex_double
 #include <complex.h>
-#define lapack_complex_double   double _Complex
+#define lapack_complex_double   std::complex<double>
 #endif
 
 #ifndef lapack_complex_double_real
